stickyuploads/__init__.py

```diff
@@ -1,3 +1,3 @@
 "Enhanced file input widget for Django which uploads the file in the background and retains value on form errors."
 
-__version__ = '0.6.1'
+__version__ = '0.7.0'
```

Comment:
Sticky uploads has been around for a long time and this version breaks some major backwards compatibility. As good a time as any to cut a 1.0